### PR TITLE
Update Release Notes for 2023.06.0

### PR DESCRIPTION
**Pull request by zachhannum**
### Intent

Update release notes for 2023.06.0.

### CODE REVIEW

File: docs/news/index.qmd [hunk 1]
```diff
@@ -9,6 +9,95 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2023.06.0
+
+**"Mountain Hydrangea"**
+
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:15]: Missing the date of release here. Check Cherry Blossom entry for an example

File: docs/news/index.qmd [hunk 2]
```diff
@@ -9,6 +9,95 @@ format:
...
+### New
+
+#### RStudio IDE
+- Updated to Electron 23.3.0 (#13030)
+- Moved Help panel font size setting to Appearance tab in Global Options (#12816)
+- Update openssl to 1.1.1t for Windows (rstudio/rstudio-pro#3675)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:21]: We have inconsistencies between Upgraded/Updated and Update. I think that we need to pick one and stick to it.

File: docs/news/index.qmd [hunk 3]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Improve visibility of focus rectangles on Server / Workbench Sign In page [Accessibility] (#12846)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:22]: The same type of thing here. If we are going to use Updated, we should use Improved.

Or, we switch things over to "Improves, Adds, Updates, Fixes"

File: docs/news/index.qmd [hunk 4]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Added ability to enable minidump generation for Electron crashes (#13025)
+- Added Quarto for Linux arm64 (#12411)
+- Updated zlib to 1.2.13 (rstudio/rstudio-pro#4531)
+
+#### Posit Workbench
+- Added the `session-connections-block-suspend` session option, controlling whether active connections can block suspension of an R session.
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:28]: I think we have been dropping ending punctuation.

Comment (AshleyHenry15) [inline docs/news/index.qmd:28]:
```suggestion
- Added the `session-connections-block-suspend` session option, controlling whether active connections can block the suspension of an R session
```

File: docs/news/index.qmd [hunk 5]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Added the `session-external-pointers-block-suspend` session option, controlling whether R objects containing external pointers can block suspension of an R session.
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:29]:
```suggestion
- Added the `session-external-pointers-block-suspend` session option, controlling whether R objects containing external pointers can block suspension of an R session
```

File: docs/news/index.qmd [hunk 6]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Added http server request processing statistics for the rserver, logged as info/debug messages, controlled by the option `www-stats-monitor-seconds` in rserver.conf (rstudio/rstudio-pro#4554).
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:30]:
```suggestion
- Added http server request processing statistics for the rserver, logged as info/debug messages, controlled by the option `www-stats-monitor-seconds` in rserver.conf (rstudio/rstudio-pro#4554)
```

File: docs/news/index.qmd [hunk 7]
```diff
@@ -9,6 +9,95 @@ format:
...
+
+### Fixed
+
+#### RStudio IDE
+- Fixed display problems with Choose R dialog when UI language is French (#12717)
+- Fixed focus switching to Help Pane search box after executing ? in the console [Accessibility] (#12741)
+- Fixed initial focus placement in Help Pane [Accessibility] (#10600)
+- Fixed invalid element role on session-suspended icon [Accessibility] (#12449)
+- Improve screen-reader support for Console pane toolbar [Accessibility] (#12825)
+- Background script jobs are now run using the global environment. This fixes the behaviour of `source()` in backgrounds jobs. (#11866)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:46]:
```suggestion
- Background script jobs are now run using the global environment. This fixes the behavior of `source()` in backgrounds jobs (#11866)
```

File: docs/news/index.qmd [hunk 8]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Fixed bug that caused Update Available dialog to show after the user selected to Ignore Update (rstudio/rstudio-pro#4179)
+- Fixed bug that prevented updating to the latest release if version was previously ignored (#12874)
+- RStudio no longer uses `reg.exe` when attempting to enumerate R versions in the Windows registry (#12599)
+- Fixed file-type icons not displaying in Finder on Mac (#12252)
+- Fixed saving and restoring window location when maximized or partially offscreen (#12463)
+- Fixed display of macOS message dialogs (#12928)
+- Set theme of menu bar, title bar, and dialogs (dark vs. light) based on RStudio theme (#12247)
+- Fixed issues with mouse back / forward navigation in Source pane, Help pane (#12932)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:54]:
```suggestion
- Fixed issues with mouse back / forward navigation in Source pane and Help pane (#12932)
```

File: docs/news/index.qmd [hunk 9]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Fixed opening files from command-line with relative paths (#12495, #12563)
+- Fixed issue with column preview with older versions of the 'pillar' package. (#12863)
+- Fixed bug where the OK button was disabled in Choose R dialog when only one version of R installed (#12916)
+- Improved robustness when chosing custom R version in Windows desktop (#12969)
+- Fixed bug that prevented RStudio Desktop from starting on Linux if desktop.ini was unreadable (#12963)
+- Improved RStudio Desktop startup behavior when user state folder inaccessible (#12988)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:60]:
```suggestion
- Improved RStudio Desktop startup behavior when the user state folder is inaccessible (#12988)
```

File: docs/news/index.qmd [hunk 10]
```diff
@@ -9,6 +9,95 @@ format:
...
+- Fixed saving files to UNC paths on Windows (#12652, #12935)
+- Fixed window disappearing when restoring from maximized on macOS (#13010)
+- Fixed Project Build Tools preferences pane showing &mdash text in French UI (#11134)
+- Fixed inserting cross references in a Quarto document showing no references (#12882)
+- Fixed keyboard shortcut help not displaying in Emacs or Sublime Text mode (#11376)
+- Fixed issue with rendering development documentation with R 4.3.0 (#12945)
+- Fixed issue with use of "Whole word" with Find in Files search (#13017)
+- Fixed issue with diagnostics freezing session with documents containing pipebind operator (#13091)
+- Fixed issue with stopping a Quarto Shiny doc render on Windows (#12546)
+- Fixed issue causing sessions to fail to start if repos contained NA value (#12946)
```
Comment (AshleyHenry15) [inline docs/news/index.qmd:70]: The next entry has NA as `NA` values - should these be the same or are they ok as is?

**Review by AshleyHenry15 — APPROVED** [review]
Some fairly minor comments - the big thing that is missing was the date.

**Review by AshleyHenry15 — APPROVED** [review]
Some fairly minor comments - the big thing that is missing was the date.

**Review by AshleyHenry15 — APPROVED** [review]
Some fairly minor comments - the big thing that is missing was the date.

File: docs/news/index.qmd (hunk 10 above)
Comment (zachhannum) [reply docs/news/index.qmd]: Probably best to have it be consistent.

**Review by mikebessuille — APPROVED** [review]
lgtm!  Ship it

**Review by mikebessuille — APPROVED** [review]
lgtm!  Ship it

**Pull request merged**